doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,322 @@
+.. _ncs_release_notes_100:
+
+|NCS| v1.0.0 Release Notes
+##########################
+
+This project is hosted by Nordic Semiconductor to demonstrate the integration of Nordic SoC support in open source projects, like MCUBoot and the Zephyr RTOS, with libraries and source code for low-power wireless applications.
```

Comment:
Suggest:
"nRF Connect SDK delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.  It includes the MCUBoot and the Zephyr RTOS open source projects which are continuously integrated and re-distributed with the SDK."

Reply:
done